product_docs/docs/pem/9/considerations/pem_pgbouncer/configuring_pgBouncer.mdx

```diff
@@ -7,17 +7,20 @@ redirects:
     - /pem/latest/pem_pgbouncer/03_configuring_pgBouncer/
 ---
 
-You must configure PgBouncer to work with the PEM database server. This example runs PgBouncer as the enterprisedb system user and outlines the process of configuring pgBouncer version 1.9 or later.
+You must configure PgBouncer to work with the PEM database server. This example runs PgBouncer as the enterprisedb system user and outlines the process of configuring pgBouncer version `<x.x>` or later.
 
 1.  Open a terminal window and navigate to the pgBouncer directory.
```

Comment:
```suggestion
1.  Open a terminal window and navigate to the PgBouncer directory.
```
Is the etc directory a subdirectory to the "PgBouncer" directory? If so, it might be clearer to saying "PgBouncer installation directory" above.

Reply:
we mean /etc/edb/pgbouncer as pgbouncer directory